glib/src/value.rs

```diff
@@ -783,6 +851,12 @@ macro_rules! numeric {
             type Checker = GenericValueTypeChecker<Self>;
 
             unsafe fn from_value(value: &'a Value) -> Self {
+                FromOwnedValue::from_value_owned(value)
+            }
+        }
+
+        unsafe impl FromOwnedValue for $name {
```

Comment:
Please also shortly grep in the sources to see if there are any manual `FromValue` impls left that should get this one too

Reply:
the only one left are Value & BoxedValue. Should they also get a FromOwnedValue impl?

Reply:
Yes